mmv1/templates/tgc/resource_converters.go.erb

```diff
@@ -41,6 +41,7 @@ func ResourceConverters() map[string][]cai.ResourceConverter {
 		"google_compute_forwarding_rule":                  {compute.ResourceConverterComputeForwardingRule()},
 		"google_compute_global_address":                   {compute.ResourceConverterComputeGlobalAddress()},
 		"google_compute_global_forwarding_rule":           {compute.ResourceConverterComputeGlobalForwardingRule()},
+		"google_compute_health_check":           {compute.ResourceConverterComputeHealthCheck()},
```

Comment:
nit: try to align these in future entries! We run `gofmt` on the output files, but the source should be as close as possible as well.